debian/control

```diff
@@ -6,7 +6,8 @@ Build-Depends: debhelper-compat (= 13),
  pkg-config,
  qtbase5-dev,
  qtwebengine5-dev,
- libkiwix-dev (>= 11.0.0~)
+ libkiwix-dev (>= 13.0.0), libkiwix-dev (<< 14.0.0)
+ libzim-dev (>= 9.0.0), libzim-dev (<< 10.0.0)
```

Comment:
Debian packaging seems to not understand `libkiwix-dev (<< 14.0.0)`. (See https://github.com/kiwix/kiwix-desktop/actions/runs/8053475026/job/21995971665)

Maybe this should be a simple `<` ?

Reply:
I checked the [documentation](https://www.debian.org/doc/debian-policy/ch-relationships.html#syntax-of-relationship-fields) during my review. `<<` seems to be the only valid syntax:

> The relations allowed are `<<`, `<=`, `=`, `>=` and `>>` for strictly earlier, earlier or equal, exactly equal, later or equal and strictly later, respectively.

Reply:
I think the problem is with the missing comma after `libkiwix-dev (<< 14.0.0)`

Reply:
Fixed!